consensus/src/dag/types.rs

```diff
@@ -46,6 +46,7 @@ struct NodeWithoutDigest<'a> {
     timestamp: u64,
     payload: &'a Payload,
     parents: &'a Vec<NodeCertificate>,
+    shares: &'a Vec<u8>, // serialized bytes of randomness shares
```

Comment:
This should be an `Option<>`?

Reply:
Also, annotate with `#[serde(with = "serde_bytes")]` (lessons from last week 😬 )